mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -61,20 +90,24 @@ def binary_cross_entropy(pred,
                          reduction='mean',
                          avg_factor=None,
                          class_weight=None,
-                         ignore_index=255):
+                         ignore_index=255,
+                         avg_non_ignore=False,
+                         **kwargs):
     """Calculate the binary CrossEntropy loss.
 
     Args:
         pred (torch.Tensor): The prediction with shape (N, 1).
         label (torch.Tensor): The learning label of the prediction.
+            Note: In bce loss, label < 0 is invalid.
         weight (torch.Tensor, optional): Sample-wise loss weight.
         reduction (str, optional): The method used to reduce the loss.
             Options are "none", "mean" and "sum".
         avg_factor (int, optional): Average factor that is used to average
             the loss. Defaults to None.
         class_weight (list[float], optional): The weight for each class.
-        ignore_index (int | None): The label index to be ignored. Default: 255
-
+        ignore_index (int | None): The label index to be ignored. Default: 255.
```

Comment:
The default value of `ignore_index` is different in `cross_entropy`, `binary_cross_entropy` and `mask_cross_entropy`. The default behavior should be the same since we provide a unified API to call these three functions in the CrossEntropyLoss class.

Reply:
And the default value 255 will cause unexpected results when the number of classes is larger than 255.

Reply:
> And the default value 255 will cause unexpected results when the number of classes is larger than 255.

What about all set `Default: -100`? or maybe `Default: -255`?

Reply:
> What about all set `Default: -100`? or maybe `Default: -255`?

PyTorch uses -100 as the default value.